addons/point_of_sale/static/src/app/navbar/closing_popup/closing_popup.js

```diff
@@ -60,14 +62,20 @@ export class ClosePosPopup extends Component {
             return;
         }
         if (this.hasUserAuthority()) {
-            this.dialog.add(CancelConfirmationDialog, {
-                title: _t("Payments Difference"),
-                body: _t(
-                    "The money counted doesn't match what we expected. Want to log the difference for the books?"
-                ),
-                confirm: this.closeSession.bind(this),
-                cancel: () => {},
-            });
+            const response = await ask(
+                this.dialog,
+                {
+                    title: _t("Payments Difference"),
+                    body: _t(
+                        "The money counted doesn't match what we expected. Want to log the difference for the books?"
+                    ),
+                },
+                {},
+                CancelConfirmationDialog
```

Comment:
Just remove cancel confirmation dialog, it's useless. use the default confirmation dialog and make "proceed anyway" and make it the "cancel"